Plots/Masking/NCL_mask_5.py

```diff
@@ -0,0 +1,142 @@
+"""
+NCL_mask_5.py
+=============
+This script illustrates the following concepts:
+    - Using cartopy.feature options to display map information
+    - Paneling two plots on a page with 'subplots' command
+    - Drawing contours
+    - Explicitly setting the fill colors for contours
+    - Drawing contours over land only
+    - Using draw order resources to mask areas in a plot
+    - Adding a label bar
+    - Implementing best practices when choosing a color scheme 
```

Comment:
As we have done with other plots, we can link the color examples that are already in the gallery. You could do something like "For more information on colormap choices, refer to [these examples](https://geocat-examples.readthedocs.io/en/latest/gallery/index.html#colors)"